[Home] 솔루션 내 인터랙션 추가

kelvin-chihyun opened the pull request:
## 🔍️ 이 PR을 통해 해결하려는 문제가 무엇인가요?

> 어떤 기능을 구현한건지, 이슈 대응이라면 어떤 이슈인지 PR이 열리게 된 계기와 목적을 Reviewer 들이 쉽게 이해할 수 있도록 적어 주세요
> 일감 백로그 링크나 다이어그램, 피그마를 첨부해도 좋아요

- 피드백에 따라, 솔루션 컴포넌트 내 각 솔루션 클릭시 이미지에 인터랙션을 추가하였습니다.

## ✨ 이 PR에서 핵심적으로 변경된 사항은 무엇일까요?

> 문제를 해결하면서 주요하게 변경된 사항들을 적어 주세요

![animation](https://github.com/ReOpenSG/ReOpenSG/assets/101504272/4eabb3e4-d107-4b19-9bb9-97e6a10a5243)


## 🔖 핵심 변경 사항 외에 추가적으로 변경된 부분이 있나요?

> 없으면 "없음" 이라고 기재해 주세요

-

## 🙏 Reviewer 분들이 이런 부분을 신경써서 봐 주시면 좋겠어요

> 개발 과정에서 다른 분들의 의견은 어떠한지 궁금했거나 크로스 체크가 필요하다고 느껴진 코드가 있다면 남겨주세요

-

## 🩺 이 PR에서 테스트 혹은 검증이 필요한 부분이 있을까요?

> 테스트가 필요한 항목이나 테스트 코드가 추가되었다면 함께 적어주세요

-

### 📌 PR 진행 시 이러한 점들을 참고해 주세요

- Reviewer 분들은 코드 리뷰 시 좋은 코드의 방향을 제시하되, 코드 수정을 강제하지 말아 주세요.
- Reviewer 분들은 좋은 코드를 발견한 경우, 칭찬과 격려를 아끼지 말아 주세요.
- Review는 특수한 케이스가 아니면 Reviewer로 지정된 시점 기준으로 3일 이내에 진행해 주세요.
- Comment 작성 시 Prefix로 P1, P2, P3 를 적어 주시면 Assignee가 보다 명확하게 Comment에 대해 대응할 수 있어요
  - P1 : 꼭 반영해 주세요 (Request Changes) - 이슈가 발생하거나 취약점이 발견되는 케이스 등
  - P2 : 반영을 적극적으로 고려해 주시면 좋을 것 같아요 (Comment)
  - P3 : 이런 방법도 있을 것 같아요~ 등의 사소한 의견입니다 (Chore)

#

---

## 📝 Assignee를 위한 CheckList

- [ ] To-Do Item

kelvin-chihyun committed:
➕ [Add] : 파트너사 이미지 추가

빠진 파트너사 CI 이미지를 추가하였습니다.

Related to #189

kelvin-chihyun committed:
🌈 [Design] : 추가한 파트너사 이미지 레이아웃에 배치

빠졌던 파트너사 이미지를 추가하여 배치하였습니다.

Related to #189

kelvin-chihyun committed:
🌈 [Design] : fade-in 애니메이션 추가

Solution 컴포넌트 요소 별로 애니메이션을 적용하기 위해,fade-in 속성을 정의하였습니다.

kelvin-chihyun committed:
🐛 [Fix] : fade-in animation 효과 수정

피드백에 따라 fade-in animation 일부 변경하였습니다.

kelvin-chihyun committed:
✨ [Feat] : 메인 페이지 - 솔루션 인터랙션 추가

메인 페이지 - 솔루션 컴포넌트의 솔루션이 바뀔때마다,
이미지에 인터랙션을 추가하였습니다.

kelvin-chihyun committed:
🏛️ [Style] : 애니메이션 효과 이름 변경

애니메이션 속성이 변하며, 이름을 변경하였습니다.
fade-in -> bounceFade

ingbinsee reviewed (approved):
P2: 제품 페이지에서 이미지 수정이 있었음에 따라 메인 페이지에서도 같은 이미지로 수정 요청드립니다.

<img width="1251" alt="image" src="https://github.com/ReOpenSG/ReOpenSG/assets/140426866/bc7c1bf8-50b1-4752-a7f3-d659b3d1f3f9">

kelvin-chihyun committed:
🌈 [Design] : 메인페이지 - Industries css 일부 수정

텍스트 색상을 open-foundation-teritary에서 white로,
aos-delay시간을 150 + index * 100 => 150 + index * 50으로 수정

kelvin-chihyun committed:
🤝 [Merge] : 원격 저장소와 로컬 저장소 동기화

ryudg reviewed (approved):
변경 사항 확인했습니다!!! 🙌

kelvin-chihyun committed:
📂 [Rename] : 이미지 변경

이미지 수정에 따른 이미지 이름을 변경하였습니다.

kelvin-chihyun committed:
🐛 [Fix] : 경로 수정

경로가 바뀜에따라, '/machines/Lift Type'에서,
'/machines/Lift Type1'로 수정하였습니다.
